packages/nimble-components/src/combobox/index.ts

```diff
@@ -70,6 +70,26 @@ export class Combobox
     @observable
     public controlWrapper?: HTMLElement;
 
+    public override get value(): string {
+        return super.value;
+    }
+
+    // This override is to work around an issue in FAST where an old filter value
+    // is used after programmatically setting the value property.
+    // See: https://github.com/microsoft/fast/issues/6749
+    public override set value(next: string) {
+        super.value = next;
+        if (!this.valueUpdatedByInput) {
+            // Using index notation to avoid error from accessing private member
+            // eslint-disable-next-line @typescript-eslint/dot-notation
+            this['filter'] = '';
```

Comment:
Would be nicer if we could use `@ts-expect-error` for a specific error instead of accessing as an indexed property but that's not possible yet so this is fine since it's commented  https://github.com/microsoft/TypeScript/issues/19139